docs/how-to-guides/pro-development.md

```diff
@@ -0,0 +1,44 @@
+# develop on the pro platform
+
+Here is how to develop for the pro platform using docker.
+
+It suppose [you already have setup docker for dev](../introduction/dev-environment-quick-start-guide.md).
+
+You should have two kind of shell:
+- the shell for openfoodfacts
+- the shell for openfoodfacts-pro, this is a shell where you have source the `setenv-pro.sh`,
```

Comment:
I think it will be quite a burden to maintain a .env for producer platform just to tweak several variables, that's why I propose the script approach for now.

In the future we may change that, but maybe this would mean generating a .env from two files ? (maybe thanks to an entry in the Makefile ?)

With the script, the Makefile works (environment variables have precedence over the .env).